### ISSUE TITLE
Dll redirection

### ISSUE THREAD

**Issue by Carlos-Rocha** [wqweto/UMMM#10]
Hi Vladimir

I need to use regular DLLs redirection in my manifests, so I added a new (very simple) function to UMMM to create entries like

`<file name="md5.dll" loadFrom="bin\md5.dll"></file>`

from the line in INI

`Dll bin\md5.dll`

I couldn't find how to create a "pull request" so here are the changes, in case you are interested:

Just add to pvProcess another Case:
```
Case "dll"
    pvDumpDll At(vRow, 1), cOutput
```
and the function
```
Private Function pvDumpDll(ByVal sPath As String, cOutput As Collection) As Boolean
    Const FUNC_NAME     As String = "pvDumpDll"
    Dim a() As String
    On Error GoTo EH
    If LenB(sPath) <> 0 Then
        a = Split(sPath, "\")
        cOutput.Add Printf("    <file name=""%1"" loadFrom=""%2""></file>", a(UBound(a)), sPath)
    End If
    Exit Function
EH:
    PrintError FUNC_NAME
    Resume Next
End Function
```

**Comment by wqweto** [comment]
Implemented in 1.0.12 release

**Comment by Carlos-Rocha** [comment]
Your solution is even better, thank you
